contributors/devel/sig-architecture/conformance-tests.md

```diff
@@ -66,9 +66,14 @@ Examples of features which are not currently eligible for conformance tests:
 Examples of tests which are not eligible for promotion to conformance:
 - anything that checks specific Events are generated, as we make no guarantees
   about the contents of events, nor their delivery
+    - If a test depends on events it is recommended to change the test to 
```

Comment:
I think it would be helpful to add a little context, something like:

Conformance tests are intended to be stable and backwards compatible according to the standard API deprecation policies. Therefore any test that relies on specific output that is not subject to the deprecation policy cannot be promoted to conformance. Examples of tests which are not eligible...

Reply:
+1, I'll hold for other feedback and update.

Reply:
Done.